components/title/basic/src/index.scss

```diff
@@ -0,0 +1,48 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$c-title-basic: $c-white;
+$c-subtitle-basic: $c-white;
+
+$ff-roboto-sans-serif: 'Roboto', sans-serif; // Will be moved to nc-theme-basic
```

Comment:
I have simplified the use of variables in this component, in order to be aligned with the theme-basic _typography definition (h1 and h2) and _settings (font-family, font-size and font-weight).
I will just maintain the variables for the color of the title and subtitle, which can be overwritten in each basic-theme as required.